services/user.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        ** info
+) -> None:
+    get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        **info
+    )
+
+
+def get_user(user_id: int) -> User:
+    return get_user_model().objects.get(id=user_id)
+
+
+def update_user(
+        user_id: int,
+        username: str = None,
+        password: str = False,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> None:
+    user_to_update = get_user_model().objects.get(id=user_id)
+
+    if username:
```

Comment:
```suggestion
    if username is not None:
```
please, fix everywhere

Reply:
due to zen of Python 
"Explicit is better than implicit."